ui/src/stylesheets/globals/_typography.scss

```diff
@@ -91,8 +91,8 @@ h6 {
 
 code {
   background-color: #f5f6fe;
-  color: var(--core-violet-4);
-  border-radius: .3em;
+  color: var(--color-code);
+  border-radius: 3px;
```

Comment:
Is there a reason why you prefer `px` to `em`? I generally use `em` so that magnitudes scale automatically in certain scenarios, like the user specifying a custom font-size in their browser for accessibility reasons.